tests/FVTTests/README.md

```diff
@@ -0,0 +1,35 @@
+# Explorer-JES-fvt

```

Comment:
may be add instructions to run tests here, all mention we need live backend system

Reply:
Have updated the readme with the correct command